gmd.py

```
@@ -8,22 +8,103 @@
+                att_count = len(res.document.attachments)
+                att_bytes = res.document.metadata.get("attachmentBytes", 0) or 0
+                tokens = res.document.stats.get("totalTokensApprox", 0) or 0
+                table.add_row(
+                    res.markdown_path.name,
+                    str(att_count),
+                    f"{att_bytes / (1024 * 1024):.2f}" if att_bytes else "0.00",
+                    str(tokens),
+                )
+            ui.console.print(table)
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)